### ISSUE TITLE
Behaviour of `toHaveRendered()` and `toNotHaveRendered` with no parameters

### ISSUE THREAD

**Issue by dglsparsons** [PsychoLlama/expect-enzyme#22]
This seems particularly unintuitive - 
`toHaveRendered` with no args implies that _something_ has been rendered - so should pass as long as the component did not render `null`
`toNotHaveRendered` with no args implies that _nothing_ has been rendered - so should fail unless the component rendered `null`. 

Current behaviour is that `toHaveRendered` with no args will _always_ fail, and `toNotHaveRendered` with no args will _always_ pass.

**Comment by PsychoLlama** [comment]
Implemented as `v1.2.0` ([see release comment](https://github.com/PsychoLlama/expect-enzyme/issues/18#issuecomment-328401089)). Thanks for suggesting this!

**Comment by dglsparsons** [comment]
This doesn't seem to be working correctly.

Simple example: 

```
it("should fail", () => {
    let wrapper = shallow(<div/>);
    expect(wrapper).toNotHaveRendered(); // it has rendered a div, so should fail.
});
```

Expected behaviour would be for the test to fail, but instead it passes. 

This is with expect-enzyme `v1.2.0`

**Comment by PsychoLlama** [comment]
Oh dear. I'll get a patch out hopefully tonight.

**Comment by PsychoLlama** [comment]
Okay, fixed in `v1.2.1`. For realz this time.
